src/sonic-yang-models/yang-models/sonic-flex_counter.yang

```diff
@@ -39,6 +57,9 @@ module sonic-flex_counter {
                 leaf FLEX_COUNTER_DELAY_STATUS {
                     type flex_delay_status;
                 }
+                leaf POLL_INTERVAL {
+                    type poll_interval_range_1s_30s;
```

Comment:
>poll_interval_range_1s_30s [](http://example.com/codeflow?start=25&length=26)

Is there a technical reason that the maximal is 30s? How about just using non-zero uint32?

Reply:
@qiluo-msft 
It was suggested in the comments to use ranges from this file: https://github.com/Azure/sonic-utilities/blob/master/counterpoll/main.py#L26
When the PR was opened, this range was from 1 to 30 seconds. But two weeks ago, the maximum was increased to the 60 seconds.
About using non-zero uint32: Do you mean using the whole range for uint32: range 1..4294967295?
Is this correct?

Reply:
Yes that's my question. How do you think?

Reply:
As far as I understand, the POLL_INTERVAL range is limited by the CLI. For example - https://github.com/Azure/sonic-utilities/blob/master/counterpoll/main.py#L94.
I think we can have the same range here for consistency. But we should limit the lower bound.
As for me we can use not limiting the upper bound of the polling interval, however the lower bound is useful. Setting the counters polling interval to < 100ms will produce a huge load on the CPU.
The same was written in the comments: https://github.com/Azure/sonic-buildimage/pull/9276#discussion_r750784662
So, we can use the same following range for all containers: 100..4294967295.
What do you think?

Reply:
@qiluo-msft 
Extended poll interval range to max uint32 with limiting lower bound: range 100..4294967295.
Please review.